section-engineering-education/engineering-education
# Add comments and likes to articles on author profile pages

**Purpose**:
Display the number of likes and comments for each article on the Author profile pages. I plan to test this on Production if everything else looks good. Once I can verify it's working properly on production, I'll remove the feature toggle and the other console log statements. 

This code also includes a Jenkins Pipeline for publishing branches from EngEd to beta.section.io.

**How to Test**:
1. Download and [Install Hugo](https://gohugo.io/getting-started/installing/) if you haven't already. 
2. Pull down branch and start the development environment `hugo server -D`.
3. This feature has been wrapped in a feature toggle requiring a cookie to be present. In order to view this code locally or on beta, open the developer tools and enter `document.cookie = "hyvorTalkCookie=1;Path=/"`into the console. Refresh the page. 
4. Navigate to an Authors profile page, you should see the number of comments and likes next to each article in the article list view. This is only present currently on the Author Profile pages. 
5. Feel free to choose an article and leave a comment. This will only post comments to our staging environment, so it's safe to test.
